contents/docs/self-host/runbook/clickhouse/vertical_scaling.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Scaling ClickHouse Vertically
```

Comment:
Problem: We link to this doc from deployments, but in a non-discoverable way. I wouldn't be able to find this document if I needed to, especially since we already have a configuration + troubleshooting docs separately.

Reply:
Agree, how can we make it better? The purpose of the _Runbook_ section is to contain all the _"procedures to begin, stop, supervise, and debug a PostHog infrastructure"_ that I think it is quite different than configuring it.

Reply:
Proposal - let's link to https://posthog.com/docs/self-host/runbook at the top of troubleshooting.md?